crates/ide_diagnostics/src/handlers/incorrect_case.rs

```diff
@@ -341,7 +341,6 @@ mod F {
     }
 
     #[test]
-    #[ignore]
     fn bug_trait_inside_fn() {
```

Comment:
This one I need to dig into deeper tomorrow -- now I am wholly confused about which wrong behavior is wrong here :D